release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2078,6 +2078,8 @@ $ ./openshift-install wait-for install-complete
 
 * Due to the inclusion of old images in some image indexes, running `oc adm catalog mirror` and `oc image mirror` might result in the following error: `error: unable to retrieve source image`. As a temporary workaround, you can use the `--skip-missing` option to bypass the error and continue downloading the image index. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed].
 
+* In {product-title} on {rh-openstack}, you can assign a floating IP address to a reservation port to have a predictable SNAT address for egress traffic. Currently, when an egress IP address on an {rh-openstack} network moves from one node to another because of a node failover, for example, the neutron reservation port is removed and recreated. However, when an administrative user associates a floating IP address with a neutron reservation port, the delete operation hangs due to insufficient privileges. The administrative user must unset the floating IP address association manually to resolve the issue. (link:https://issues.redhat.com/browse/OCPBUGS-4902[*OCPBUGS-4902*])
```

Comment:
Okay, so we got to say that this is about EgressIP feature [1] and the reservation port comes from that, right not it's not clear. No need to mention about the failover case, it's not relevant here. The main point is that the OpenStack user that was used to install the cluster has to have access to FIPs assigned to the reservation ports. Otherwise the EgressIPs will be undeletable.

[1] https://docs.openshift.com/container-platform/4.11/networking/ovn_kubernetes_network_provider/configuring-egress-ips-ovn.html

Reply:
@andreaskaris will you take a look at Michal's input, so I can make one change for the release deadline? Thanks!

Reply:
{code}
When using the EgressIP feature in {product-title} on {rh-openstack}, you can assign a floating IP address to a reservation port to have a predictable SNAT address for egress traffic. The floating IP association must be created with the same user that was used to install the {product-title} cluster. Otherwise any delete or move operation for the EgressIP will hang indefinitely due to insufficient privileges. A user with sufficient privileges must unset the floating IP address association manually to resolve the issue. (link:https://issues.redhat.com/browse/OCPBUGS-4902[*OCPBUGS-4902*])
{code}

Reply:
Better? @dulek

Reply:
@andreaskaris I have a draft that I will post shortly, but will wait to see comments for yours. If that is good, I will make that update on Wednesday for final review from you and Michal. Thanks!

Reply:
What @andreaskaris wrote catches it perfectly!